packages/web3-eth2-beaconchain/src/schema.ts

```diff
@@ -0,0 +1,223 @@
+import { IBaseAPISchema } from '../../web3-eth2-core/src/schema'
+
+export const DefaultSchema: IBaseAPISchema = {
+    packageName: 'eth2-beaconchain',
+    routePrefix: '/eth/v1/beacon/',
+    methods: [
+        {
+            name: 'getGenesis',
+            route: 'genesis',
+            restMethod: 'get',
+            inputFormatter: null,
+            outputFormatter: null,
+            errors: null,
+            errorPrefix: 'Failed to get genesis block:'
+        },
+        {
+            notImplemented: true,
```

Comment:
`notImplemented` is defaulted true in the interface, perhaps you might want to change the logic surrounding how this works